content/docs/command-reference/data/status.md

```diff
@@ -43,27 +43,32 @@ DVC uncommitted changes:
 (there are other changes not tracked by dvc, use "git status" to see)
 ```
 
-As shown above, the `dvc data status` displays changes in multiple categories:
-
-- _Not in cache_ indicates that the hash for files are recorded in `dvc.lock`
-  and `.dvc` files but the corresponding cache files are missing.
-- _DVC committed changes_ indicates that there are changes that are
-  `dvc-commit`-ed that differs with the last Git commit. There might be more
-  detailed state on how each of those files changed: _added_, _modified_,
-  _deleted_ and _unknown_.
```

Comment:
One question though: what's an `unknown` change? Does it apply to untracked files too? I didn't know where to mention that in the updated text.

Reply:
One question pending here cc @dberenbaum @skshetry 🙏🏼 

> what's an unknown change?

Reply:
For example, let's say you clone a repo with a tracked directory but haven't done `dvc fetch/pull`. You have an md5 in the .dvc file but the directory is empty. If you modify the directory (for example, add a file to it), DVC can tell that the directory doesn't match the md5 but has no info about the granular changes to the files in that directory.

Reply:
> DVC can tell that the directory doesn't match the md5 but has no info about the granular changes to the files in that directory

So it only applies to `--granular` right? (Assuming that generalizing that example is the one scenario for this.) I added a note in that option instead of trying to cover it up here (it still counts as "new, modified, or deleted").

Reply:
@skshetry Do you know if it can ever apply to non-granular data?

Reply:
yeah, it only applies to granular.